src/ContainerRenameModal.jsx

```diff
@@ -63,15 +67,21 @@ const ContainerRenameModal = ({ container, version, updateContainerAfterEvent })
 
     const renameContent = (
         <Form isHorizontal>
-            <FormGroup fieldId="rename-dialog-container-name" label={_("New container name")}
-                    validated={nameError ? "error" : "default"}
-                    helperTextInvalid={nameError}>
+            <FormGroup fieldId="rename-dialog-container-name" label={_("New container name")}>
                 <TextInput id="rename-dialog-container-name"
                         value={name}
-                        validated={nameError ? "error" : "default"}
                         type="text"
                         aria-label={nameError}
                         onChange={value => handleInputChange("name", value)} />
+                {nameError && (
+                    <FormHelperText>
+                        <HelperText>
+                            <HelperTextItem icon={<ExclamationCircleIcon />} variant="error">
+                                {nameError}
+                            </HelperTextItem>
+                        </HelperText>
+                    </FormHelperText>
```

Comment:
These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1266-20230413-122810-fbb171ea-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)